tuxemon/monster.py

```
@@ -638,6 +638,7 @@ def get_sprite(self, sprite: str, **kwargs: Any) -> Sprite:
             )
             if flair_path != MISSING_IMAGE:
                 flair_sprite = graphics.load_sprite(flair_path, **kwargs)
+                assert surface.image and flair_sprite.image
```

Again, I don't think this assert can ever fail.
`surface` and `flair_sprite` are in any case loaded using `graphics.load_sprite` or `graphics.load_animated_sprite` methods. These two methods should fail if the provided path is invalid.